IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -22,31 +22,31 @@
     <hr />
     <div>
         <table border="1" width="90%">
-            <tr>
-                <th>ID</th>
-                <th>Name</th>
-                <th>Brewery Type</th>
-                <th>Street</th>
-                <th>Address 1st Part</th>
-                <th>Address 2nd Part</th>
-                <th>City</th>
-                <th>State</th>
-                <th>County Province</th>
-                <th>Postal Code</th>
-                <th>Country</th>
-                <th>Longitude</th>
-                <th>Latitude</th>
-                <th>Phone</th>
-                <th>Website Url</th>
-                <th>Update At</th>
-                <th>Created At</th>
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Brewery Type</th>
+                    <th>Street</th>
+                    <th>Address 1st Part</th>
+                    <th>Address 2nd Part</th>
+                    <th>City</th>
+                    <th>State</th>
+                    <th>County Province</th>
+                    <th>Postal Code</th>
+                    <th>Country</th>
+                    <th>Longitude</th>
+                    <th>Latitude</th>
+                    <th>Phone</th>
+                    <th>Website Url</th>
+                    <th>Update At</th>
+                    <th>Created At</th>
+                </tr>
+            </thead>
 
-            </tr>
-            @{
-                foreach (Breweries.BreweryCollection brr in brew)
+            <tbody>

```

Comment:
I added the table headers into html tag and rest of the table rows in html tag to improve the readability and neatness of the code.
https://www.w3schools.com/tags/tag_thead.asp
https://www.w3schools.com/tags/tag_tbody.asp